showcase/gapic-showcase/src/test/java/com/google/showcase/v1beta1/it/ITGdch.java

```diff
@@ -0,0 +1,202 @@
+package com.google.showcase.v1beta1.it;
```

Comment:
Missing copyright headers for this file and all other new files.

Reply:
added copyright to the 3 new files